services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -170,9 +170,9 @@ const deleteIndexedBlocks = async job => {
 		const accountsByPublicKey = await getAccountsByPublicKey(generatorPkInfoArray);
 		if (accountsByPublicKey.length) await accountsDB.upsert(accountsByPublicKey, trx);
 		const forkedTransactionIDs = await getTransactionsByBlockIDs(blocks.map(b => b.id));
-		const forkedVotes = await getVotesByTransactionIDs(forkedTransactionIDs);
-		await transactionsDB.deleteIds(forkedTransactionIDs, trx);
-		await votesDB.deleteIds(forkedVotes.map(v => v.tempId), trx);
+		const forkedVotesTransactionIDs = await getVotesByTransactionIDs(forkedTransactionIDs);
+		await transactionsDB.deleteByPrimaryKey(forkedTransactionIDs, trx);
+		await votesDB.delete({ whereIn: { property: 'transactionID', values: forkedVotesTransactionIDs } }, trx);
```

Comment:
This logic should be modified as per the requested changes for the removal of the `votes_aggregates` table.